preprocessing/edit_labels.ipynb

```diff
@@ -0,0 +1,142 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Dataframe shape\n",
+      "Rows:  650000 \n",
+      "Columns:  3\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import os\n",
+    "from pathlib import Path\n",
+    "\n",
+    "# parent directory\n",
+    "parent = os.path.join(os.getcwd(), os.pardir)\n",
+    "parent_dir = os.path.abspath(parent)\n",
+    "\n",
+    "csv_path = os.path.join(parent_dir, \"dataset\", \"test.csv\")\n",
+    "test_df = pd.read_csv (filepath_or_buffer = csv_path, names = [\"star_rating\", \"headline\", \"body\"])\n",
```

Comment:
We should be consistent with our internal nomenclature for field names: you are using "headline" where I used "title," I don't care which we use but we should be consistent